### PR TITLE
Add ss_llama_simple in place of ss_llama

### PR DESCRIPTION
**Pull request by danbraunai**
## Description
**Everyone should use the new ss_llama_simple instead of ss_llama. You will need to `uv sync` after pulling this.**

Created a new ss_llama_simple model in https://github.com/goodfire-ai/simple_stories_train/tree/dev. This is the same as the regular llama but:
- We don't use flash attention
- We train in float32 instead of (mostly) bfloat16. We needed to do this to avoid divergence
- We split up kv_attn into two matrices
- We change the paths of all modules so they are similar to gpt2. The paths are now:
```
  [
    "h.*.mlp.gate_proj",
    "h.*.mlp.up_proj",
    "h.*.mlp.down_proj",
    "h.*.attn.q_proj",
    "h.*.attn.k_proj",
    "h.*.attn.v_proj",
    "h.*.attn.o_proj",
  ]
```

Removed ss_llama_config.yaml.

## Motivation and Context
- ss_llama was trained with simplestories, but was uploaded to huggingface. The model we decompose is thus the huggingface version.
- This makes it quite awkward to interpret, because the huggingface model is complex and has a lot of options.

## How Has This Been Tested?
Testing that ss_llama gives the same as ss_llama_simple with a couple of different configs:
1 (Lucius asked me for this one).
- ~SPD [Run](https://wandb.ai/goodfire/spd/runs/qwbm917v) of ss_llama_simple (This run accidentally didn't use the sigmoid fix)~
- SPD [Run](https://wandb.ai/goodfire/spd/runs/hhekh9wt) of ss_llama_simple with batch size 8 (accidental). It still gets similar enough results to the below
- SPD [Run](https://wandb.ai/goodfire/spd/runs/3rthvni3) of ss_llama

2.
- SPD [Run](https://wandb.ai/goodfire/spd/runs/g19cvhed) of ss_llama
- SPD [Run](https://wandb.ai/goodfire/spd/runs/va94mjf5) of ss_llama_simple. This is currently IMO close enough to the run above. I expect some differences due to randomness and because it was trained on a different tokenizer (4019 tokens compared to 4096).
- Also, we verified that the pretrained models had similar val losses

Relatedly, this set off a full investigation of running SPD on models trained with a different number of steps. Wandb report [here](https://wandb.ai/goodfire/spd/reports/Effect-of-target-model-training-steps-on-SPD--VmlldzoxNDgwMTEwMQ).

## Does this PR introduce a breaking change?
No

### CODE REVIEW

**Review by oli-clive-griffin — APPROVED** [review]
LGTM!
